cprover_bindings/src/goto_program/stmt.rs

```diff
@@ -21,6 +21,37 @@ use tracing::debug;
 ///     `while (c) {stmt1, stmt2}`
 ///      would translate to `Stmt::while_loop(c, vec![stmt1, stmt2], loc)`
 /// Statements can also be created using the converters in the `Expr` module.
+///
+#[derive(Copy, Debug, Clone)]
+pub enum PropertyClass {
```

Comment:
I like the idea of having an enum, but this code should live inside kani-compiler codegen module.

Reply:
A bit hard to do that when the `Location` type its used in is in `cprover_bindings`. IMO leave it, it's certainly not the only "Kani thing" in this crate.

Reply:
This shouldn't be hard though. The Location should still work over strings since the CProver API accepts any string.

Codegen module should have a method that builds the property correctly from the `enum` value.

Reply:
Ah, right, fair.

Reply:
Will move it to codegen

Reply:
Tried moving to codegen but it presented problems for the assert_stmt implementation. As the Assert is ultimately a Stmt and the StmtBody needs to be implemented, we need to retain the enum in the stmt.rs file.

Would refactoring this later be acceptable @celinval ?